src/ghdcbot/core/interfaces.py

```
@@ -1,8 +1,9 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import Iterable, Protocol, Sequence
+from typing import Any, Iterable, Protocol, Sequence
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Iterable`, `Sequence`

Import from `collections.abc`

(UP035)